sources/dicom/large_image_source_dicom/web_client/views/AssetstoreImportView.js

```diff
@@ -25,6 +42,8 @@ const AssetstoreImportView = View.extend({
             }, this).dicomwebImport({
```

Comment:
In looking at the error message, I think we should be doing `this.$('.g-validation-failed-message').html(err.responseJSON.message);` rather than `this.$('.g-validation-failed-message').html(err.responseText);`.

Reply:
Fixed!